tutorial.md

```diff
@@ -106,9 +106,9 @@ class MyToolBar extends StatelessComponent {
       ),
       child: new Row(
         children: [
-          new IconButton(icon: 'navigation/menu'),
+          new IconButton(icon: Icons.menu),
```

Comment:
You should also add `uses-material-design: true` to `flutter.yaml` and remove the sections `material-design-icons`